### ISSUE TITLE
`'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0 warning (Swift Package Manager)

### ISSUE THREAD

**Issue by Piranja** [google/GoogleSignIn-iOS#42]
Hi,

I observe a warning in the `SignInSample` example project:
```
Package AppAuth:
'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.5.99.
```
The same warning appears after adding GoogleSignIn SDK using Swift Package Manager to another project.

Can you advise how this warning could be fixed, or should I use POD instead of Swift Package?

**Comment by Haibo-Zhou** [comment]
I meet the same issue with POD, add this snipper into Podfile, the warning is gone.

```
post_install do |installer|
    installer.pods_project.targets.each do |target|
        target.build_configurations.each do |config|
        config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '9.0'
        end
    end
end
```

**Comment by Piranja** [comment]
Thanks for the answer!

Is any possibility to avoid this warning when using Swift Package Manager?

**Comment by Haibo-Zhou** [comment]
I don't use SPM because when meet issue during creating my project. The help info from internet is most about Pod.

But I think you could find the same place `config.build_settings` in SPM, and increase its deployment version to have a try.

**Comment by petea** [comment]
This warning is a known issue(openid/AppAuth-iOS#601) in the AppAuth library when used with recent versions of Xcode which no longer support iOS 8.

**Comment by tryboxx** [comment]
Will it be fixed in the near future? For SPM we don't have this option with `config.build_settings`...

**Comment by Timac** [comment]
I would love to see these warnings fixed when using Swift Package Manager. They occur with Xcode 13.0RC when targeting iOS 15.0:

<img width="1015" alt="warnings" src="https://user-images.githubusercontent.com/738543/133630817-8afe8f5d-e1d1-42a6-84e6-53481681abcf.png">

**Comment by reversepanda** [comment]
Currently this has been reported with https://github.com/openid/AppAuth-iOS but the project seems to be unmaintained for some time now. There is a PR to fix this, but I'm not seeing an development in the repo for over a year. I wonder if Google needs to fork and start maintaining a core dependency themselves.

Anyways it's just a (annoying) warning a the dependency seems to be quite mature nonetheless.

**Comment by codedbypm** [comment]
https://github.com/openid/AppAuth-iOS/pull/661

It's merged! 🎉

**Comment by CemYil03** [comment]
> [openid/AppAuth-iOS#661](https://github.com/openid/AppAuth-iOS/pull/661)
> 
> It's merged! 🎉

Hey, this is my pull request and it is not merged yet. Someone with write access needs to merge it in. At the time it is just approved :(

**Comment by petea** [comment]
This should be resolved with the release of [AppAuth for iOS 1.5.0](https://github.com/openid/AppAuth-iOS/releases/tag/1.5.0).
